conf/schema-loading-custom-values.yaml

```
@@ -25,3 +26,8 @@ schemaLoading:
   # username: <AWS_ACCESS_KEY_ID>
   # password: <AWS_ACCESS_SECRET_KEY>
   # dynamoBaseResourceUnit: 10
+
+  #
+  # General
+  #
+  #schemaType: ledger
```

It would look nicer with one space.

```suggestion
  # schemaType: ledger
```